playbooks/roles/load-balancer-v2/templates/backend.map.hcl.j2

```diff
@@ -3,4 +3,5 @@ template {
   destination      = "/etc/haproxy/backend.map"
   create_dest_dirs = false
   backup           = true
+  command          = "flock -n /tmp/haproxy-reload-lock service haproxy reload"
```

Comment:
I actually always use this syntax now, it seems to work fine :D

I realized though I need to add one thing to `flock`: `-E 0`, because not being able to take the lock shouldn't be considered a failure (default exit code is 1).